health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -1,44 +1,360 @@
+
+
+            const switchText = localIsActive ? "Active" : "Inactive";
+            return (
+              row?.ServiceRequest?.[0]?(<Switch
+                isCheckedInitially={row?.ServiceRequest?.[0]?.isActive}
+                label={switchText}
+                onToggle={toggle}
+              />):(<>{t("CHECKLIST_TOBE_CONFIGURED")}</>)
+            );
+        case "HCM_CHECKLIST_ACTION":
+          const role_code = row?.data?.role;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)